db/migrate/20240123151135_create_foods.rb

```diff
@@ -0,0 +1,12 @@
+class CreateFoods < ActiveRecord::Migration[7.1]
+  def change
+    create_table :foods do |t|
+      t.string :food
+      t.string :measurement_unit
+      t.integer :price
+
+      t.timestamps
+      t.references :user, foreign_key: true
+    end
+  end
```

Comment:
Great work @Sami-Ullah-Tufail. The linter errors(all correctable) were as a result of your IDE's 4 column indentation rule instead of 2. Could you kindly in all subsequent commits run ```robocop -a``` before pushing?